epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
```

Comment:
There are kind of two issues here - links that open in the same app as the reading system, and links that open externally. If we change this statement, we lose the recommendation that a browser based client open links in a new browser context (eg open a new window/tab, don't replace the contents of the current window/tab, or worse open the content in a subframe of the reading system). But maybe that is fine, since we do still recommend a warning in that case.